### DISCUSSION TITLE
Is the returned price inclusive of TOU charges from our Provider ?

### DISCUSSION THREAD

**Discussion by craigcurtin-dev** [amberelectric/public-api#34] [General]
As per the topic - it does not appear that it is - but the wholesale prices are now GST inclusive ?

Where do we go to the get the Static Charges (Green, Insurance etc etc) can not see them in the price feed any longer ?

Is the TOU prices included anywhere in the feed - or should we just pull them off the provider (Ausgrid) websites ?

Craig

**Comment by madpilot** [comment]
Hi Craig,

The wholesale prices (for better or worse) have always included GST.

We no longer include the static prices in the API - everything has been pre-calculated for you.

The price should also include the correct TOU charge already. If not, send an email to info@amber.com.au and I can check to see if you are attached to the right tariff in our database.

**Comment by craigcurtin-dev** [comment]
Thanks email sent

Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________

**Reply by craigcurtin-dev** [reply]
Yep all looks fine now - you can close this one off
